server/gui/cxg_server.py

```diff
@@ -0,0 +1,66 @@
+import traceback
+
+from PyQt5.QtCore import QRunnable, pyqtSlot
+
+from server.gui.utils import WorkerSignals
+
+
+class cellxgeneServer():
+    def __init__(self, parent, host="127.0.0.1", port=8000):
+        self.app = None
+        self.host = host
+        self.port = port
+
+    def setup_app(self):
+        from server.app.app import Server
+        server = Server()
```

Comment:
I find the naming confusing as there are several "servers" all doing different things (and the term is super generic).    `Server` is really a `RESTDataServer` (or something similar), as it serves up the r/o data to the front-end.   Maybe make the names more descriptive.

Also, in this context, do we need a separate `cellxgeneServer` class, or can this functionality be incorporated into `server.app.app`?

Reply:
I agree, I'll rename.

Hmm, I'll think on integrating the functionality into server.app.app. That's a good idea.

Reply:
Actually it's not just a REST data server, it's also serving the main html page and the assets. Can we postpone this naming issue to add to the `import cellxgene` ticket?

Reply:
I did actually move the functionality to server.app.app (which rippled through to server.cli.launch.py) but I think it makes it better code in general.

Reply:
yes, OK delaying on naming issue

Reply:
changed cxg_server.py -> workers.py to reflect the deletion and changed role of the file